Project-3/Project#3_tf

```diff
@@ -0,0 +1,292 @@
+
+  
+  (b)  Transitive 
+
+located_in. This property has 'transitive' listed, but should not. Its definition is that of "a relation between two independent continuants, the target and the location, in which the target is entirely within the location." The definition of 'transitive' states that "if individual x is related to individual y, and individual y is related to individual z, then individual x will be related to individual z. In other words, a single 'hop' is implied over a chain of two along a given property if that property is transitive." That definition doesn't seem to be always satisfiable in 'located_in.' The idea here seems to be that the property of 'located_in' is a relation between two independent continuants, and one of these is the target that is located_in location-x, and the other is the location itself. In simpler terms, if you find something then the thing in question is located-in the place that you found it. Nevertheless, there does not seem to be transivity at play here. Instead, one seems to find a reiteration of a previous term in more technical form, without the transitive property playing a role. To say that something, the target, is located_in the location seems to simply, redundantly, repeat that the target was located (in the location where it was located.) I am all for using repetition as a way to emphasize something in some case, and in some instances it can even carry some pragmatic inmport whereby the context adds a semantic layer to what is being reiterated. Still, repetition of the same term (location) in slightly modified language doesn't seem to entail transitivity. One can, for example, think of the military having a plan to intercept a package or weapon from the enemy, and thus plans in advance what location-x will be on a map. Location-x is where the target will be intercepted and thus located_in at the time of the interception. Then the military intercepts the package at the specified location and time: in this case, the location and the target align/overlap in spatio-temporal terms. But the target was located_in somewhere at every point, at some place/location at every step. There seems to be no real transitive property here, for to say that something is found in the place that you found it seems both redundant and more of a form of reflexivity than transitivity. The same applies to saying that something is where it is--yes, it is true that the target is at the location where the target is (located_in) at any given time... but that doesn't add any real information, it just repeats a spatially-oriented version of 'it is what it is.' One should have a hard time asserting, if willing to do so at all, that the second 'it is' in the platitude/truism carries its meaning via the transitive property from the first 'it is.' It should be the same for located_in, which should not have transitive listed.
```

Comment:
Sometimes when ontologies are large enough it's not obvious when redundant information has been added, so including properties like transitivity allows us to infer that implicit information automatically. 
In this case, I think though perhaps redundant in some sense, located in does strike me as transitive. Do you have a counterexample?